nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1475,11 +1488,19 @@ private void checkFieldInitialization(ClassTree tree, VisitorState state) {
       }
     }
     for (Element constructorElement : errorFieldsForInitializer.keySet()) {
+      List<Symbol> fieldSymbols =
+          errorFieldsForInitializer
+              .get(constructorElement)
+              .stream()
+              .map(element -> ASTHelpers.getSymbol(getTreesInstance(state).getTree(element)))
+              .filter(Objects::nonNull)
```

Comment:
How can the `Symbol` be `null` here?

Reply:
I was just being careful to avoid ```NPEs```. Please let me know if you would like to remove it ?

Reply:
9111438dfe555f7dd37913279d515d9b41400562